packages/opencensus-exporter-azure/src/azure-trace.ts

```diff
@@ -0,0 +1,135 @@
+import {
+    Exporter,
+    ExporterConfig,
+    Span,
+    Logger,
+    logger,
+    View,
+    Measure,
+    AggregationData,
+    ExporterBuffer
+} from '@opencensus/core';
+
+import * as ApplicationInsights from 'applicationinsights';
+import { IllegalOptionsError } from './azure-stats';
+
+export interface TraceParams {
+    registeredViews: View[];
+    registeredMeasures: Measure[];
+    recordedData: { [key: string]: AggregationData[] };
+}
+
+export interface AzureTraceExporterOptions extends ExporterConfig {
```

Comment:
I would suggest to keep these types in a separate file, maybe in `types.ts`?

Reply:
Please include the JSDoc comment.

Reply:
Done!